bin/gqlint.js

```diff
@@ -33,8 +37,20 @@ function initProgram() {
 function getGQLintConfig() {
   let data;
 
+  // paths to search for a config file in order of precedence
+  let paths = ['.', path.dirname(program.args[0]), os.homedir()];
+
+  for (let p of paths) {
+    let configPath = path.join(p, program.config);
+    if (fs.existsSync(configPath)) {
+      program.fullConfigPath = configPath;
+      console.error('Using config file: ' + program.fullConfigPath);
```

Comment:
Maybe this could be a console.info as it's not really an error.

Reply:
The reason that I logged it to error was that I thought logging to stdout might interfere with how editors parse the result. I am not sure how they are doing it and if an extra line there will make any difference... do you know if stdout will interfere?

Reply:
Good point, to be honest I'm not sure. We could also remove this line maybe, it doesn't add *that* much.

Reply:
it could be removed, but I have been burned by another program that didn't have this that left me scratching my head for hours as to why it wasn't working. There was hidden config file it was picking up and using which was throwing things off. 

maybe I could change it to `console.warn` since that still goes to stderr? or if you don't want that either I can just take it out. Let me know what works

Reply:
Cool, let’s do warn then :)